src/components/MapsSection/MapsSection.tsx

```diff
@@ -31,52 +41,91 @@ const MapsSection = () => {
+    };
+  }, [currentVisu, mapsData]);
+
   useEffect(() => {
     if (mapsData.length != 0) {
       setCurrentVisu(mapsData.map((map) => map.fields)[0]);
     }
-
-    setLoading(false);
   }, [mapsData]);
 
   return (
-    <Wrapper id="maps-visu">
-      <SectionTitle variation="white">Mapas e Visualizações</SectionTitle>
-      <TagsContainer>
-        {mapsData.map(({ fields: tag }: { fields: IEEInfo }) => (
-          <Tag
-            key={tag.id}
-            active={(tag.id === currentVisu.id).toString()}
-            onClick={() => updateCurrentVisu(tag.id)}
-          >
-            {capitalize(tag.name)}
-          </Tag>
-        ))}
-      </TagsContainer>
-      {loading ? (
-        <LoadingContainer>
-          <SpinningIcon
-            src={LoadingImage}
-            alt="Representação de uma flor que existe no nordete com cinco pétalas"
+    <MapSectionWrapper id="maps-visu">
+      <SectionTitle variation="black">Mapas e Visualizações</SectionTitle>
+      <BoxWrapper>
+        <PreviewWrapper>
+          <ExpandBox href={`/map?name=${currentVisu.id}`}>
+            <VisuIcon id="expand" size={20} />
+          </ExpandBox>
+          <MapPoster
+            alt=""
+            width={500}
+            height={400}
+            src={
+              typeof currentVisu.poster === "object"
+                ? `https:${currentVisu.poster.fields.file.url}`
+                : currentVisu.poster
+            }
           />
-          <span>Carregando..</span>
-        </LoadingContainer>
-      ) : (
-        <MapPoster
-          alt=""
-          width={500}
-          height={400}
-          src={
-            typeof currentVisu.poster === "object"
-              ? `https:${currentVisu.poster.fields.file.url}`
-              : currentVisu.poster
-          }
-        />
-      )}
-      <VisuName>{capitalize(currentVisu.name)}</VisuName>
-      <Description>{currentVisu.description}</Description>
-      <LinkButton href={`/map?name=${currentVisu.id}`}>Visualizar</LinkButton>
-    </Wrapper>
+        </PreviewWrapper>
+        {mapsData.length != 0 && (
+          <TagsContainer>
+            {mapsData.map(({ fields: tag }: { fields: IEEInfo }) => (
+              <TagButton
+                key={tag.id}
+                active={(tag.id === currentVisu.id).toString()}
+                onClick={(e) => updateCurrentVisu(tag.id, e)}
+              >
+                <VisuHeader>
+                  <VisuName active={(tag.id === currentVisu.id).toString()}>
+                    {capitalize(tag.name)}
+                  </VisuName>
+                  <IconWrapper>
+                    <VisuIcon
+                      active={(tag.id === currentVisu.id).toString()}
+                      id={tag.id === currentVisu.id ? "open-eye" : "closed-eye"}
+                      size={20}
+                    />
+                    <Divider />
+                    <LinkButton
+                      href={`/map?name=${currentVisu.id}`}
```

Comment:
We want it to redirect to the visualization related to the title, not the current being display on screen